src/test/java/homework23/ui/tests/WebShopUITest.java

```diff
@@ -0,0 +1,52 @@
+package homework23.ui.tests;
+
+import homework23.api.tests.WebShopApiTest;
+import homework23.ui.CartPage;
+import homework23.ui.CheckOutPage;
+import homework23.ui.utils.BaseTest;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static homework23.ui.utils.HelpFunc.*;
+import static homework23.ui.utils.HelpFunc.getOrderOn;
+
+public class WebShopUITest extends BaseTest {
+
+    @Test()
+    public void buyAndCheckItemInCartAndCheckOut() {
+
+        WebShopApiTest.createUser();
+        String userName = WebShopApiTest.createdUser.getUsername();
+        String password = WebShopApiTest.createdUser.getPassword();
```

Comment:
should not be dependent on test method.
It's better to create a separate user using api inside current test.